### PR TITLE
Document loanbook aggregation

### PR DESCRIPTION
**Pull request by jacobvjk**
- add documentation of exposure-weighted aggregate alignment metric on the loan book level
- publish documentation with pkgdown
- add some interpretation for the company level metrics

### CODE REVIEW

**Comment by jacobvjk** [comment]
@jdhoffa I followed the `pkgdown` documentation [here](https://pkgdown.r-lib.org/articles/pkgdown.html) for a basic attempt at publishing the vignettes on a GH page. I works locally, but on the feature branch, I do not see the website. Not sure if this is expected and only works once it is merged into `main`, but maybe you know more about it?

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```
Comment (jdhoffa) [inline .github/workflows/pkgdown.yaml:7]: @jacobvjk see here, the website is only deployed when merged to master or main

File: vignettes/company_alignment_metric.Rmd [hunk 2]
```diff
@@ -117,7 +119,7 @@ This must be true because every technology $a$ must have one and only one direct
...
 
-**TODO**
+Previously, it was mentioned that the aggregated value of $y_{b,c,t}$ does not indicate where the sources of (mis)alignment can be found. The disaggregation into $BO$ and $PO$ drivers of the company aggregate alignment metric aims to provide a high level overview of just that. In essence, $y_{b,c,d,t}$ only adds up the technology deviations for either buildout or phaseout technologies, while still dividing by the sector level scenario value of the company (including all technologies). This means that we get a simple disaggregation of the overall alignment metric into its sum parts. Hence, it should not be read as an indicator in isolation. However, the disaggregation can be very helpful in determining if the overall alignment is driven by a lack of buildout or by slow phaseout. Additionally, what may look like overall alignment near the target with $y_{b,c,t}$ close to zero may in theory be composed of significantly overaligned buildout and significantly misaligned phaseout or vice versa. This goes to show that, companies active in sectors with both buidlout and phaseout technologies, it is important to understnad the drivers behind that overall alignment metric.
```
Comment (jdhoffa) [inline vignettes/company_alignment_metric.Rmd:122]: Typo: `understnad`

**Review by jdhoffa — APPROVED** [review]
I think this LGTM. 

I would suggest merging and, once it is on gh pages and in a more human readable format, having Nick review.

File: vignettes/company_alignment_metric.Rmd (hunk 2 above)
Comment (jacobvjk) [reply vignettes/company_alignment_metric.Rmd]: thanks, fixed

**Comment by jacobvjk** [comment]
> I think this LGTM.
> 
> I would suggest merging and, once it is on gh pages and in a more human readable format, having Nick review.

absolutely, all recent documentation still needs a round of review/approval by Nick

**Comment by jdhoffa** [comment]
Ah can I ask something after the fact? 

Can you use this as the pkgdown template instead of flatly?
https://github.com/RMI-PACTA/pacta.pkgdown.template

**Pull request merged**